android/app/src/main/java/org/uwaterloo/subletr/pages/createaccount/CreateAccountPageUiState.kt

```diff
@@ -0,0 +1,20 @@
+package org.uwaterloo.subletr.pages.createaccount
+
+sealed interface CreateAccountPageUiState {
+	object Loading : CreateAccountPageUiState
+
+	enum class Gender(val gender: String) {
```

Comment:
Can you make this separate from this UiState?
I think we'll need it elsewhere too.